README.md

```diff
@@ -0,0 +1,208 @@
+# Mitmproxy configuration for Pretendo
+
+This repo contains configurations, scripts, and certificates for using
+`mitmproxy`/`mitmweb`/`mitmdump` to intercept traffic from Nintendo consoles,
+including the Wii U and the 3DS. It supports multiple operation modes, including
+redirecting requests to a local Pretendo Network server and collecting Wii U and
+3DS network dumps.
+
+## Collecting network dumps
+
+1. Download and install Docker using the
+   [official guide](https://docs.docker.com/get-docker/).
+2. First, make sure to **disable** Inkay or Nimbus to ensure that you are
+   connected to the official Nintendo Network servers. Then, download the right
+   NoSSL patches for your console.
+   <!-- TODO: Where are the patches? Link them here. -->
```

Comment:
I wanted to address this, are the necessary patches in a private repo? Or are users supposed to use the ones included in Inkay/Nimbus?

Reply:
Our public patches do disable SSL verification but they also handle redirection, so they aren't suitable for proxy use alone

For the 3DS, the SSL patch can be found in the fork used for this PR made by @InternalLoss https://github.com/SciresM/3DS-SSL-Patch/pull/2

For the Wii U, the SSL patch is in a private repo on our org. Truth be told, I don't remember why we went with a private repo? I'll ask around and see if I can figure out why we did that

It might be a good idea to just have the patches in this repo though for easier access, since the only real time you'd want an SSL patch is when using a proxy server. InternalLoss is part of our team now, so I'm sure he wouldn't have an issue with us hosting it here. And the Wii U patch was made by us originally anyway

Reply:
While the repo is closed source, I have been known to give out the precompiled patch to people who need it. So for now, I've just uploaded it to Google Drive https://drive.google.com/file/d/14IM--ZcY5cEGNYR30dAgszZKvjjflkuR/view?usp=drive_link

Feel free to download it and just include it in this repo. Same with the IPS patch, I was given permission to just put the patch here in this repo

After this is merged we can deal with the logistics of the private repo and all that on our side

Reply:
I can't access the Google Drive link.

Reply:
Goes to show how much I use Google Drive. This one should work? https://drive.google.com/file/d/14IM--ZcY5cEGNYR30dAgszZKvjjflkuR/view?usp=sharing

Reply:
Got it

Reply:
For context this is an Aroma setup module, so it goes in `sd:/environments/[ENV]/modules/setup`